packages/mapviewer/tests/cypress/tests-e2e/drawing.cy.js

```diff
@@ -972,9 +1003,13 @@ describe('Drawing module tests', () => {
             })
 
             cy.get(`[data-cy^="button-remove-layer-"]`).click()
+
             cy.readStoreValue('state.layers.activeLayers').should((layers) => {
-                expect(layers).to.be.an('Array').lengthOf(0)
+                expect(layers).to.be.an('Array').and.to.have.length(0)
             })
+
+            cy.get(`[data-cy^="button-remove-layer-"]`).should('not.exist')
```

Comment:
you don't specify any layer ID here (it ends abruptly at `remove-layer-`... is it wanted?